test/sonatype-nexus3.test.ts

```diff
@@ -105,3 +122,27 @@ function overrideStackWithContextDomainName(app: cdk.App, stack: cdk.Stack,
   });
   return { app, stack };
 }
+
+
+describe('Test stack context', () => {
+  let app: cdk.App;
+  let stack: cdk.Stack;
+
+  test('Throw error if domainName is not specified', () => {
```

Comment:
Let's merge this case into existing test suite.

You can improve the existing initialization method to support creating app with different contexts.